config/locales/cz.yml

```diff
@@ -80,106 +80,107 @@ cz:
+        CrimethInc. Je odbojová allince a decentralizovaná síť která se zavázala k anonymní kolektivní akci a útěku z vězení našeho času. Snažíme se znovuobjevit náš život ve světe s ohledem na principy sebeurčení a vzájemné pomoci.
 
-        We believe that you should be free to dispose of your limitless potential on your own terms: that no government, market, or ideology should be able to dictate what your life can be. If you agree, _let’s do something about it._
-      more_link: More about CrimethInc. →
-      donate_button_text: Support this site
+        Věříme že by si měl mít možnost volně využít svůj nekonečný potenciál tak jak uznáš za vhodné. Žádná vláda, trh , nebo ideologie by ti neměla diktovat jaký bude tvůj život. Jestli souhlasíš, _Tak s tím pojďme něco udělat._
+      more_link: Víc o CrimethInc. →
+      donate_button_text: Podpoř nás.
 
     contact:
       newsletter:
-        heading: Email Updates
-        description: Sign up here to receive regular email updates. _(Processed by Mailchimp.)_
-        signup_form_placeholder: What’s your email address?
-        signup_form_select_placeholder: You want a lot of emails? Or a little?
-        signup_form_select_option_lot:    Every little thing. (Many emails)
-        signup_form_select_option_little: Just the big stuff! (Fewer emails)
+        heading: Aktualizace Emailem
+        description: Přihlaš se pro odběr pravidelných zpráv z našeho webu. _(Zpracováno přes MailChimp.)_
+        signup_form_placeholder: Jaký je tvůj email?
+        signup_form_select_placeholder: Chceš hodně emailů? Nebo málo?
+        signup_form_select_option_lot:    Každou maličkost! (Many emails)
+        signup_form_select_option_little:  Jen to důležité! (Fewer emails)
         signup_button_text: Sign Up
       social:
-        heading: Find Us Elsewhere
-        description: Keep up with our activities on the following additional platforms.
+        heading: Sleduj nás jinde
+        description:  Sleduj naše aktivity na následujících platformách.
 
     mantra:
-      - If your heart is free, the ground you stand on is liberated territory. Defend it.
-      - Only those prepared to go too far will learn how far they can go.
-      - Desperate measures are long overdue.
-      - Cradle the seed, even in the volcano’s mouth.
-      - The future is in chains within you — set it free.
-      - A spark for every powder keg.
+      - Jestli je tvé srdce volné, zem na které stojíš je již osvobozená. Nenech si ji vzít.
+      - Jen ti připraveni dojít příliš daleko zjistí jak daleko mohou jít 
+      - Zoufalé časy už dávno skončily
+      - Starej se o semínko, i ve chřtánu sopky.
+      - Budoucnost uvnitř tebe je v řetězech — osvoboď ji.
+      - Jiskra pro tvuj střelný prach.
       - Esperanza no espera.
       - Mobilis in mobili
+      - Bojujeme další válku.
```

Comment:
is this one a czech saying? Just wondering

Reply:
Nah, nothing that interesting i'am afraid :/ , It's just loosely translated to czech language.